AptSpringAPI/src/main/java/com/salesforce/aptspring/Verified.java

```diff
@@ -54,26 +55,26 @@
  *  <li>@Bean annotations define at least one name for the bean.</li>
```

Comment:
Do you think we should have a bit of context as to why each of these things is important?

Reply:
I think that is what the readme is for.   I wanted a concise list of checks in the annotation.   Could link out to something though.